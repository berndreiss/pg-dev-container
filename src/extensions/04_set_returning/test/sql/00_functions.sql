SELECT create_integer_set(NULL);
SELECT create_integer_set(-1);

SELECT create_integer_set(5);
SELECT * FROM create_integer_set(5);
SELECT * FROM create_integer_set(5) LIMIT 2;


SELECT * FROM get_programming_languages();
SELECT name FROM get_programming_languages() WHERE year = 1995;