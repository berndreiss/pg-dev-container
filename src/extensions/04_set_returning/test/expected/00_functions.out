SELECT create_integer_set(NULL);
 create_integer_set 
--------------------
(0 rows)

SELECT create_integer_set(-1);
ERROR:  elements parameter must be greater than 0
SELECT create_integer_set(5);
 create_integer_set 
--------------------
                  0
                  1
                  2
                  3
                  4
(5 rows)

SELECT * FROM create_integer_set(5);
 create_integer_set 
--------------------
                  0
                  1
                  2
                  3
                  4
(5 rows)

SELECT * FROM create_integer_set(5) LIMIT 2;
 create_integer_set 
--------------------
                  0
                  1
(2 rows)

SELECT * FROM get_programming_languages();
  name  |     inventor      | year 
--------+-------------------+------
 C      | Dennis Ritchie    | 1972
 C++    | Bjarne Stroustrup | 1985
 Python | Guido van Rossum  | 1991
 Java   | James Gosling     | 1995
(4 rows)

SELECT name FROM get_programming_languages() WHERE year = 1995;
 name 
------
 Java
(1 row)

